tests/PHPC-bug00631.phpt

```diff
@@ -0,0 +1,42 @@
+--TEST--
+PHPC-631: UTCDateTime::toDateTime() may return object that cannot be serialized
+--SKIPIF--
+<?php if (!defined("HHVM_VERSION_ID")) exit("skip HHVM uses 'UTC' instead of '+00:00'"); ?>
```

Comment:
Should the skip message be changed? This reads as if you're skipping it on HHVM (despite the conditional logic).

Perhaps say `PHP uses '+00:00' instead of 'UTC'`. Alternatively, could we keep the test compatible with both by using some EXPECTF regex?